Makefile

```diff
@@ -847,7 +847,7 @@ $(RCPTS)/fvtr.rcpt: $(RCPTS)/package.rcpt
 	   AT_WD=$(AT_WD) \
 	   AT_PREVIOUS_VERSION=$(AT_PREVIOUS_VERSION) \
 	   AT_DIR_NAME=$(AT_DIR_NAME) \
-	   ./fvtr.sh -f \
+	   ./fvtr.sh -f -d \
```

Comment:
I don't think this is a good idea.
It's going to print too much unnecessary data.